saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SemanticRoleLabeling/SRLMultiGraphDataModel.scala

```diff
@@ -36,19 +39,27 @@ class SRLMultiGraphDataModel(parseViewName: String = null, frameManager: SRLFram
 
   val tokens = node[Constituent]((x: Constituent) => x.getTextAnnotation.getCorpusId + ":" + x.getTextAnnotation.getId + ":" + x.getSpan)
 
+  // Edges
+
   val sentencesToStringTree = edge(sentences, stringTree)
   val sentencesToTokens = edge(sentences, tokens)
   val sentencesToRelations = edge(sentences, relations)
   val relationsToPredicates = edge(relations, predicates)
   val relationsToArguments = edge(relations, arguments)
 
+  // Sensors
+
   sentencesToTokens.addSensor(CommonSensors.textAnnotationToTokens _)
   sentencesToRelations.addSensor(textAnnotationToRelation _)
-  sentencesToRelations.addSensor(textAnnotationToRelationMatch _)
+
+  // TODO - Verify if we really need this matching sensor. This is very inefficient.
+  //  sentencesToRelations.addSensor(textAnnotationToRelationMatch _)
```

Comment:
Okay. Let me have a look again. I ran `aTr`, `bTr`, `cTr` and `dTr` and the results were similar to the ones in the readme. Can check again.

Will investigate the `matching` sensor issue.